share/completions/netctl-auto.fish

```diff
@@ -0,0 +1,33 @@
+set -l cmds list store restore stop-all start stop restart switch-to status enable disable reenable is-enabled edit
+
+function __fish_print_netctl-auto_profile
+    for line in (netctl-auto list)
+        set -l profile (string sub -s 3 $line)
```

Comment:
Where does the "3" come from? Does it print whitespace before the actual profile? If so, `string trim` might be nicer.